### ISSUE TITLE
Dependency error on install with Vagrant 1.9.7 on MacOS Sierra

### ISSUE THREAD

**Issue by gizmola** [dotless-de/vagrant-vbguest#263]
I upgraded to Vagrant 1.9.7, and now I'm unable to install vbguest.

Here's what I'm getting:

```
Installing the 'vagrant-vbguest' plugin. This can take a few minutes...
Bundler, the underlying system Vagrant uses to install plugins,
reported an error. The error is shown below. These errors are usually
caused by misconfigured plugin installations or transient network
issues. The error from Bundler is:

conflicting dependencies winrm-fs (= 1.0.1) and winrm-fs (= 0.2.0)
  Activated winrm-fs-0.2.0
  which does not match conflicting dependency (= 1.0.1)

  Conflicting dependency chains:
    winrm-fs (= 0.2.0), 0.2.0 activated

  versus:
    winrm-fs (= 1.0.1)

  Gems matching winrm-fs (= 1.0.1):
    winrm-fs-1.0.1
```

**Comment by fnordfish** [comment]
vagrant-vbguest has no (direct) dependency on `winrm-fs`, but `vagrant` has. I suspect you've upgraded from <= 1.9.5 to 1.9.7 - in which case vagrant updated it's internal ruby, which is a usual reason of plugin related issues.
Try:

```bash
vagrant plugin list
# note of your installed plugins 
rm -rf ~/.vagrant.d/{gems,plugins.json}
# re-install all your plugins
```

If you are still having problems, please head over to mitchellh/vagrant

**Comment by gizmola** [comment]
For others who might encounter this issue, I just wanted to comment.
 
Removing plugin files and settings as suggested were not beneficial.  I spent a bunch of time googling and getting into the weeds of ruby, but the eventual fix ended up being pretty easy.

I simply  that I had to uninstall vagrant and reinstall it.  In the process of doing so, it fixed up the dependency problems.

[The manual page is here.](https://www.vagrantup.com/docs/installation/uninstallation.html)

Those instructions for OS X are:
```
rm -rf /Applications/Vagrant
rm -f /usr/local/bin/vagrant
sudo pkgutil --forget com.vagrant.vagrant
```
